pkg/util/shell/cat.go

```diff
@@ -0,0 +1,44 @@
+package shell
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+)
+
+func Cat(arguments []string, stdin io.Reader, stdout io.Writer) error {
```

Comment:
use the interp context here instead of those parameters please to keep the command implementation more generic